package.json

```diff
@@ -4,24 +4,32 @@
   "dependencies": {
     "babel-core": "^6.23.0",
     "babel-loader": "^6.4.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
```

Comment:
Is this and the dependency below to support SFE and/or paragon? I think I'm likely missing something, if they're a dependency of those packages, are they needed here?

Reply:
I think that's why they're here; will confirm that they're actually needed and not just experimental leftovers

Reply:
Everything seems to be working fine after removing these deps, I'll delete them.

Reply:
I checked out the error and it seems like maybe these are in fact needed. Error:
```
16:02:30 Module build failed: ReferenceError: Unknown plugin "transform-object-rest-spread" specified in "/home/jenkins/workspace/edx-platform-accessibility-pr/edx-platform/node_modules/@edx/studio-frontend/package.json" at 0, attempted to resolve relative to "/home/jenkins/workspace/edx-platform-accessibility-pr/edx-platform/node_modules/@edx/studio-frontend"
```
I do find it a bit confusing why they're needed because [SFE already specifies it as a dependency](https://github.com/edx/studio-frontend/blob/master/package.json#L34). Am I wrong in thinking that we shouldn't have to require it here too?

Reply:
Oh wait, I think I get it. Because we are transpiling SFE, we need to include these babel packages and thus need to include them, right? It seems a bit frail, but I suppose we might not have too many transpiling packages.